syntax = "proto3";
package msplayer;
import "common.proto";
option cc_generic_services = true;

message TeamTestRequest
{
}

message TeamTestResponse
{
	ClientError error = 1;
}


service PlayerTeamService
{
  rpc EnterScene (TeamTestRequest) returns (TeamTestResponse);
}